packages/core/src/app/common/error/SentryErrorLogger.ts

```diff
@@ -135,6 +135,12 @@ export default class SentryErrorLogger implements ErrorLogger {
     }
 
     private handleBeforeSend: (event: Event, hint?: EventHint) => Event | null = (event, hint) => {
+        if (
+            event.breadcrumbs?.filter((breadcrumb) => breadcrumb.data?.url?.includes('convertcart'))
```

Comment:
I don't think we should filter out errors by hardcoding third party URLs like this, because it's impossible to filter out all the third party URLs defined by custom checkouts.

I think we have the following filter function that only tracks exceptions if they are thrown by the code that we own:
https://github.com/bigcommerce/checkout-js/blob/68a899bc56c59b298681d9872c7ff462f107af4b/packages/core/src/app/common/error/SentryErrorLogger.ts#L114-L135

It seems it's not working as expected if the exception only has a single frame in the stacktrace, and the frame is the Sentry logger itself. i.e.:

<img width="926" alt="Screen Shot 2022-11-14 at 9 22 38 am" src="https://user-images.githubusercontent.com/667603/201547578-808d738a-e6f9-4f2c-8fae-70f8c16ada84.png">

I think a better fix is to actually update the filter function to also cater for that particular scenario. Thoughts?

Reply:
Thanks DC for the feedback.
I have created [CHECKOUT-7100](https://bigcommercecloud.atlassian.net/browse/CHECKOUT-7100) to follow up on this.
cc @animesh1987